build.gradle

```diff
@@ -130,20 +135,73 @@ dependencies {
 
     // These dependencies are only needed for running tests
     testCompile group: 'junit', name: 'junit', version: '4.11'
-    testCompile group: 'org.mockito', name: 'mockito-all', version: '1.10.19'
+    testCompile group: 'org.mockito', name: 'mockito-core', version: '2.26.0'
+    testCompile group: 'org.mockito', name: 'mockito-inline', version: '2.26.0' // For final classes
 
-    // Don't upgrade either of these until https://github.com/powermock/powermock/issues/717 is resolved
-    testCompile 'org.powermock:powermock-module-junit4:1.6.4'
-    testCompile 'org.powermock:powermock-api-mockito:1.6.4'
+    testCompile 'org.powermock:powermock-module-junit4:2.0.4'
+    testCompile 'org.powermock:powermock-api-mockito2:2.0.2'
 }
 
 // Set the expected module Java level (can use a higher Java to run, but should not use features from a higher Java)
-sourceCompatibility = 1.8
-targetCompatibility = 1.8
+sourceCompatibility = 1.11
+targetCompatibility = 1.11
+
+javafx {
+    version = "11.0.2"
+    modules = [
+            'javafx.graphics',
+            'javafx.fxml',
+            'javafx.web']
+}
+
+jlink {
+    launcher {
+        name = 'launcher'
```

Comment:
Why are you setting this? Isn't `project.name` a good name? 

See https://badass-jlink-plugin.beryx.org/releases/latest/#_launcher